app/src/main/java/org/wildstang/wildrank/androidv2/activities/SyncActivity.java

```diff
@@ -50,10 +60,37 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     private void beginSync() {
-        if (!SyncUtilities.isFlashDriveConnected()) {
-            showDriveNotMountedWarning();
+        if (SyncUtilities.useSAF()) {
+            SyncUtilities.openFileChooser(this);
```

Comment:
Currently, yes.

I believe that it will return you to the same folder (so you just hit the file and OK) if you plug in the flash drive before syncing (or use a non-removable device, such as the Google Drive app).